weave/panels_py/panel_observability.py

```diff
@@ -226,33 +259,59 @@ def observability(
         domain_x=user_zoom_range,
     )
 
+    runs_grouped = weave.ops.List.groupby(queued_time_data, lambda row: row["trace_id"])
+    runs_mapped = weave.ops.List.map(
+        runs_grouped,
+        lambda row: weave.ops.dict_(
+            **{
+                "trace_id": row["trace_id"][0],
+                "entity_name": row["entity_name"][0],
+                "project_name": row["project_name"][0],
+                "run_id": row["run_id"][-1],
+                "job": row["job"][-1],
+                "enqueued": row[timestamp_col_name][0],
+                "run_start": row[timestamp_col_name][-1],
+                "duration": weave.ops.Number.__mul__(
+                    weave.ops.timedelta_total_seconds(
+                        weave.ops.datetime_sub(
+                            row[timestamp_col_name].max(), row[timestamp_col_name].min()
+                        ),
+                    ),
+                    1000,
```

Comment:
this multiplication is required, because `timedelta_total_seconds` does not actually return seconds here....